src/__tests__/Navbar.test.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable*/
+
+import renderer from 'react-test-renderer';
+import Navbar from '../components/Navbar';
+import { BrowserRouter } from 'react-router-dom';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <BrowserRouter>
+        <Navbar />
+      </BrowserRouter>
+    )
+    .toJSON();
```

Comment:
- [ ] Excellent use of Jest snapshots testing 👏🏻  But you need to use Jest snapshots to test all the components. Actually, I only see "Calculator.test.js.snap and Navbar.test.js"  Please make sure you add Jest snapshots for testing these components as:  `App.js, Home.js, Quotes.js, and Key.js`. 👨🏻‍🎨